website/docs/plugins/ray_launcher.md

```diff
@@ -190,6 +190,13 @@ Loaded cached provider configuration
 ```
 </details>
 
+<details><summary>Dockerized app</summary>
+
+```commandline
+python docker_app.py --multirun hydra/launcher=ray_aws task=1,2,3
+```
```

Comment:
Since this is collapsed, it doesnt hurt to add more output related to running on a docker here :)